hil-test/tests/dma_mem2mem.rs

```diff
@@ -17,10 +17,18 @@ use hil_test as _;
 const DATA_SIZE: usize = 1024 * 10;
 
 cfg_if::cfg_if! {
-    if #[cfg(any(feature = "esp32c2", feature = "esp32c3", feature = "esp32s3"))] {
+    if #[cfg(any(esp32c2, esp32c6, esp32h2))] {
+        type DmaPeripheralType = esp_hal::peripherals::MEM2MEM1;
+    } else {
         type DmaPeripheralType = esp_hal::peripherals::SPI2;
+    }
+}
+
+cfg_if::cfg_if! {
+    if #[cfg(any(esp32, esp32s2))] {
```

Comment:
ESP32 / S2 are not supported - no need for this

Reply:
not yet!